samples/dnn/trackers/siamrpnpp.py

```diff
@@ -38,62 +42,40 @@ def track(self, x):
         self.search_net.setInput(x)
         outNames = self.search_net.getUnconnectedOutLayersNames()
         xfs_1, xfs_2, xfs_3 = self.search_net.forward(outNames)
-        self.rpn_head.setInput(np.stack([self.zfs_1, self.zfs_2, self.zfs_3]), 'input_1')
+        self.rpn_head.setInput(np.stack([self.zfs_1, self.zfs_2, self.zfs_3]),
+                               'input_1')
```

Comment:
Please avoid such line break changes

Reply:
Fixed, this is probably resulted by 80 charter length limit following PEP-8